instrumentation/events/lttng-module/net.h

```diff
@@ -258,6 +412,14 @@ LTTNG_TRACEPOINT_EVENT_CLASS(net_dev_template,
 					/* For any other header type, there is nothing to do. */
 					break;
 				}
+
+				/* Copy the transport header */
+				if (__get_transport_header_type(skb) == TH_TCP) {
+					ctf_array_type(uint8_t, tcp_hdr(skb),
+							sizeof(struct tcphdr))
+					ctf_align(uint8_t)
```

Comment:
So since the TCP struct has a 32 bits field and is part of the network variant, do I need to ctf_align(uint32_t) before the ctf_array_type of the network header and maybe add that if in both cases above for ipv4 and ipv6 (without the align)?